examples/user_guide/Network_Graphs.ipynb

```diff
@@ -347,23 +351,17 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%opts Nodes Graph [width=800 height=800 xaxis=None yaxis=None]"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "%%opts Graph [color_index='circle']\n",
-    "%%opts Graph (node_size=10 edge_line_width=1)\n",
+    "kwargs = dict(width=800, height=800, xaxis=None, yaxis=None)\n",
+    "opts.defaults(opts.Nodes(**kwargs), opts.Graph(**kwargs))\n",
```

Comment:
Either the keywords should be inlined here, or in this case make explicit options objects with the builders (assuming there are lots of `kwargs`).

Reply:
Don't agree with inlining, this avoids duplication. Don't quite follow the second suggestion.